tests/components/fronius/test_config_flow.py

```diff
@@ -261,3 +267,45 @@ async def test_import(hass, aioclient_mock):
         "host": MOCK_HOST,
         "is_logger": True,
     }
+
+
+async def test_dhcp(hass, aioclient_mock):
+    """Test starting a flow from discovery."""
+    with patch("asyncio.sleep"), patch(
```

Comment:
You should patch either the contant (if you create it) or patch the "local" asyncio.sleep:
`patch("homeassistant.components.fronius.asyncio.sleep")`